com.unity.netcode.gameobjects/Components/NetworkRigidbody2D.cs

```diff
@@ -32,7 +32,7 @@ private void Awake()
 
         private void FixedUpdate()
         {
-            if (NetworkManager.IsListening)
+            if (IsSpawned)
```

Comment:
Yes it is.  There are scenarios where NetworkManager can be null and fixed update will be invoked still for a few frames.  Either case, it is better to check the IsSpawned than NetworkManager.IsListening since NetworkManager.IsListening is the second to last thing to be set during shutdown.  Really, if you look over the Shutdown sequence you can see that by the time that is set everything else is shutdown and NetworkObjects have been despawned (depending upon whether it is an in-scene placed NetworkObject or a dynamically spawned/generated one determines how long the parent GameObject will persist).